src/upload/catalog/controller/extension/payment/libwebpay/telemetry/PluginVersion.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+class PluginVersion
+{
+    protected $soapUri = 'http://www.cumbregroup.com/tbk-webservice/PluginVersion.php?wsdl';
+    protected $client;
+
+    const ENV_INTEGRATION = 'INTEGRACION';
+    const ENV_PRODUCTION = 'PRODUCCION';
+    const PRODUCT_WEBPAY = 5;
+
+    const ECOMMERCE_WOOCOMMERCE = 1;
```

Comment:
Why do we need this extra constants?

Reply:
To represent the available values that the API accepts on that parameter.